srcpkgs/0ad/template

```diff
@@ -1,63 +1,106 @@
 # Template file for '0ad'
 pkgname=0ad
-version=0.0.26
-revision=9
-archs="x86_64* i686* aarch64* armv7l* ppc64le*"
+version=0.27.0
+revision=1
+archs="x86_64* i686* aarch64* armv7l* ppc64le* riscv64*"
+# Unclear whether ppc64le and riscv64 will work, though upstream seems to support them
 build_helper=rust
-hostmakedepends="pkg-config perl cmake python3.11 rust cargo yasm tar clang"
+_cxxtest_version=4.4
+_fcollada_version=28209
+_nvtt_version=28209
+hostmakedepends="pkg-config perl cmake python3 rust cargo yasm llvm xz m4
+ premake5 subversion"
 makedepends="SDL2-devel boost-devel gloox-devel libcurl-devel libenet-devel
  libopenal-devel libpng-devel libsodium-devel libxml2-devel miniupnpc-devel
- nspr-devel wxWidgets-gtk3-devel fmt-devel freetype-devel"
-depends="${pkgname}-data"
+ nspr-devel wxWidgets-gtk3-devel fmt-devel freetype-devel mozjs115-devel"
+depends="0ad-data"
+checkdepends="0ad-data"
 short_desc="Historically-based real-time strategy game"
 maintainer="Helmut Pozimski <helmut@pozimski.eu>"
 license="GPL-2.0-or-later, CC-BY-SA-3.0, MPL-2.0, MIT"
 homepage="https://play0ad.com"
-distfiles="https://releases.wildfiregames.com/${pkgname}-${version}-alpha-unix-build.tar.xz
- https://github.com/mozilla/gecko-dev/commit/74641307d32a59806b75cd2b8c7161aca50d5cb7.patch>1654457-virtualenv.patch"
-checksum="2e1c6df7e3312e77c5f82788664cffc3a78d3bf60606c00039275e1d13c0ee4b
- f18fa3adae7acb07a49148354b6804028e6c9d7344991622fb7c00f638173e03"
-nocross="uses bundled third-party libraries that do not cross-compile"
+distfiles="
+https://releases.wildfiregames.com/${pkgname}-${version}-unix-build.tar.xz
+https://github.com/CxxTest/cxxtest/archive/refs/tags/${_cxxtest_version}.tar.gz>cxxtest-${_cxxtest_version}.tar.gz"
+checksum="
+aa94857009750d5f61dbf016bc150e3bdcbdb3acdfc8ad20b73ab8b43e9a1ba6
+1c154fef91c65dbf1cd4519af7ade70a61d85a923b6e0c0b007dc7f4895cf7d8"
+skip_extraction="cxxtest-${_cxxtest_version}.tar.gz"
 lib32disabled=yes
 
-skip_extraction="1654457-virtualenv.patch"
-
-CXXFLAGS="-fpermissive"
+CXXFLAGS+="-fpermissive"
 # Use BFD linker to avoid erroneous detection of llvm pr8927 with *-musl
-LDFLAGS="-fuse-ld=bfd"
+LDFLAGS+="-fuse-ld=bfd"
```

Comment:
`*FLAGS` don't need to be `+=`, they are concatenated with internal defaults already